flink/v1.20/flink-runtime/NOTICE

```diff
@@ -20,7 +103,143 @@ This binary artifact includes Apache ORC with the following in its NOTICE file:
 
 --------------------------------------------------------------------------------
 
-This binary artifact includes Airlift Aircompressor with the following in its
+This binary artifact includes Apache Parquet 1.15.0 with the following in its NOTICE file:
+
+| Apache Parquet Java
+| Copyright 2014-2024 The Apache Software Foundation
+|
+| This product includes software developed at
+| The Apache Software Foundation (http://www.apache.org/).
+|
+| --------------------------------------------------------------------------------
+|
+| This product includes parquet-tools, initially developed at ARRIS, Inc. with
```

Comment:
Parquet-tools hasn't been released since 1.11.2: https://mvnrepository.com/artifact/org.apache.parquet/parquet-tools

Reply:
Let's fix this upstream as well: https://github.com/apache/parquet-java/pull/3140

Reply:
Everything starting with | is the content of the dependency NOTICE

Reply:
Yeah I think for now we should just copy the content of the NOTICE as is for the particular version of the dependency we are bundling. If there's some issue in the upstream project's notice, we can fix that there and update our NOTICE after the dependency is released?

Reply:
I'm fine with that, just pointing out that we don't strictly follow the rules of how to compose a NOTICE. For the bundled products, I think it is less error-prone to have scripts copy the whole NOTICE, instead of having to curate the NOTICES ourselves. Previously we did everything by hand